controller/kubernetes_node_controller.go

```diff
@@ -293,18 +293,17 @@ func (knc *KubernetesNodeController) syncDefaultDisks(node *longhorn.Node) (err
 		return nil
 	}
 	val = strings.ToLower(val)
-
+	storageReservedPercentageForDefaultDisk, err := knc.ds.GetSettingAsInt(types.SettingNameStorageReservedPercentageForDefaultDisk)
+	if err != nil {
+		return err
+	}
 	disks := map[string]longhorn.DiskSpec{} // nolint: ineffassign,staticcheck
 	switch val {
 	case types.NodeCreateDefaultDiskLabelValueTrue:
 		dataPath, err := knc.ds.GetSettingValueExisted(types.SettingNameDefaultDataPath)
 		if err != nil {
 			return err
 		}
-		storageReservedPercentageForDefaultDisk, err := knc.ds.GetSettingAsInt(types.SettingNameStorageReservedPercentageForDefaultDisk)
-		if err != nil {
-			return err
-		}
 		disks, err = types.CreateDefaultDisk(dataPath, storageReservedPercentageForDefaultDisk)
```

Comment:
`CreateDefaultDisk()` also needs to update the block-type disk'`StorageReserved`. Right?

Reply:
CreateDefaultDisk() only supports filesystem type. If node.longhorn.io/create-default-disk is set to true, it uses the settings.default-data-path to create a default disk, which is exclusively filesystem-based. Therefore, StorageReserved for block-type disks is not required in this context.

Reply:
Doesn't CreateDefaultDisk() support filesystem type as well?
https://github.com/longhorn/longhorn-manager/blob/601b04bebbf77ad0e98a80e3d0b82b745b994fee/types/types.go#L1122-L1134

Reply:
Yes, that's correct.  It seems that adding StorageReserved configuration to CreateDefaultDisk is necessary. I will work on modifying it to support this functionality.